Makefile

```diff
@@ -177,6 +170,6 @@ clean:
 	docker rm clair || true
 	rm -rf clair-scanner || true
 	rm -rf clair-scanner-logs || true
-	rm -rf test-results/* || true
+	rm -rf test-results || true
```

Comment:
Don't do this, it'll blow away the !.gitnore file for the subdir.  This dir should always be there, and we only want to remove the non-hidden contents of the dir.

Reply:
why do we need the test-results directory to persist? What's the point of having the empty directory in the repository?

Reply:
You don't know what order someone locally is going to run things, and what make targets they'll use.  If I for example run clair without tests, then the clair make point is required to create the entire directory structure.  The management of what needs to run setup and cleanup just becomes cumbersome to manage, vs, leaving a set location all items are expected to write to.  As tests / jobs / options expand, it provides a set location that people will look to and use without needing to be instructed, and prevents them from creating a digergance and making their own folders.